lib/indocker/repositories/clonner.rb

```diff
@@ -9,9 +9,41 @@ def initialize(configuration, logger)
   def clone(session, repository)
     raise ArgumenError.new("only git repositories should be clonned") if !repository.is_git?
 
-    session.exec!("rm -rf #{repository.clone_path} && mkdir -p #{repository.clone_path}")
-    
-    git_command = "git clone -b #{repository.branch} --depth 1 #{repository.remote_url} #{repository.clone_path}"
+    repository_remote_url = session.exec!(
```

Comment:
we should move this part to a separate method, e.g. "repository_already_clonned?(repository)". it will better describe why we do this here and also allow future improvements if needed.

Reply:
👍